[2025-04-15T07:22:06.287+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-15T07:22:06.398+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-14T07:59:00+00:00 [queued]>
[2025-04-15T07:22:06.465+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-14T07:59:00+00:00 [queued]>
[2025-04-15T07:22:06.470+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-15T07:22:06.563+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-04-14 07:59:00+00:00
[2025-04-15T07:22:06.594+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3559) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-15T07:22:06.605+0000] {standard_task_runner.py:72} INFO - Started process 3577 to run task
[2025-04-15T07:22:06.610+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'scheduled__2025-04-14T07:59:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmpq6_ezl42']
[2025-04-15T07:22:06.618+0000] {standard_task_runner.py:105} INFO - Job 82: Subtask run_dbt_trusted
[2025-04-15T07:22:06.955+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-14T07:59:00+00:00 [running]> on host c0ac39583bd8
[2025-04-15T07:22:07.499+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-04-14T07:59:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-14T07:59:00+00:00'
[2025-04-15T07:22:07.505+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-15T07:22:07.604+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-15T07:22:07.610+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-04-15T07:22:07.657+0000] {subprocess.py:86} INFO - Output:
[2025-04-15T07:22:07.668+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-04-15T07:22:07.672+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-04-15T07:37:14.292+0000] {job.py:229} INFO - Heartbeat recovered after 902.59 seconds
[2025-04-15T07:37:16.599+0000] {subprocess.py:93} INFO - [0m07:37:16  Running with dbt=1.8.0
[2025-04-15T07:37:17.090+0000] {subprocess.py:93} INFO - [0m07:37:17  Registered adapter: postgres=1.8.2
[2025-04-15T07:37:17.624+0000] {subprocess.py:93} INFO - [0m07:37:17  Found 9 models, 10 sources, 428 macros
[2025-04-15T07:37:17.628+0000] {subprocess.py:93} INFO - [0m07:37:17
[2025-04-15T07:37:17.778+0000] {subprocess.py:93} INFO - [0m07:37:17  Concurrency: 1 threads (target='dev')
[2025-04-15T07:37:17.779+0000] {subprocess.py:93} INFO - [0m07:37:17
[2025-04-15T07:37:17.793+0000] {subprocess.py:93} INFO - [0m07:37:17  1 of 4 START sql table model lead_trusted.non_pii_data ......................... [RUN]
[2025-04-15T07:53:06.264+0000] {job.py:229} INFO - Heartbeat recovered after 931.59 seconds
[2025-04-15T08:07:24.912+0000] {job.py:229} INFO - Heartbeat recovered after 812.34 seconds
[2025-04-15T08:23:12.722+0000] {job.py:229} INFO - Heartbeat recovered after 901.34 seconds
[2025-04-15T08:38:38.970+0000] {job.py:229} INFO - Heartbeat recovered after 905.73 seconds
[2025-04-15T08:38:44.834+0000] {subprocess.py:93} INFO - [0m08:38:44  1 of 4 OK created sql table model lead_trusted.non_pii_data .................... [[32mSELECT 6879790[0m in 3687.04s]
[2025-04-15T08:38:44.837+0000] {subprocess.py:93} INFO - [0m08:38:44  2 of 4 START sql table model lead_trusted.payment_data ......................... [RUN]
[2025-04-15T08:54:29.769+0000] {job.py:229} INFO - Heartbeat recovered after 930.18 seconds
[2025-04-15T08:54:30.539+0000] {subprocess.py:93} INFO - [0m08:54:30  2 of 4 OK created sql table model lead_trusted.payment_data .................... [[32mSELECT 1212982[0m in 945.69s]
[2025-04-15T08:54:30.549+0000] {subprocess.py:93} INFO - [0m08:54:30  3 of 4 START sql table model lead_trusted.pii_data ............................. [RUN]
[2025-04-15T09:08:27.098+0000] {job.py:229} INFO - Heartbeat recovered after 795.42 seconds
[2025-04-15T09:08:37.122+0000] {subprocess.py:93} INFO - [0m09:08:37  3 of 4 OK created sql table model lead_trusted.pii_data ........................ [[32mSELECT 6879790[0m in 846.54s]
[2025-04-15T09:08:37.195+0000] {subprocess.py:93} INFO - [0m09:08:37  4 of 4 START sql table model lead_trusted.technical_data ....................... [RUN]
[2025-04-15T09:24:41.378+0000] {job.py:229} INFO - Heartbeat recovered after 927.28 seconds
[2025-04-15T09:24:44.676+0000] {subprocess.py:93} INFO - [0m09:24:44  4 of 4 OK created sql table model lead_trusted.technical_data .................. [[32mSELECT 1212982[0m in 967.47s]
[2025-04-15T09:24:44.695+0000] {subprocess.py:93} INFO - [0m09:24:44
[2025-04-15T09:24:44.696+0000] {subprocess.py:93} INFO - [0m09:24:44  Finished running 4 table models in 1 hours 47 minutes and 27.07 seconds (6447.07s).
[2025-04-15T09:24:44.765+0000] {subprocess.py:93} INFO - [0m09:24:44
[2025-04-15T09:24:44.766+0000] {subprocess.py:93} INFO - [0m09:24:44  [32mCompleted successfully[0m
[2025-04-15T09:24:44.767+0000] {subprocess.py:93} INFO - [0m09:24:44
[2025-04-15T09:24:44.769+0000] {subprocess.py:93} INFO - [0m09:24:44  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-04-15T09:24:45.669+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-15T09:24:45.714+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-15T09:24:45.715+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=scheduled__2025-04-14T07:59:00+00:00, execution_date=20250414T075900, start_date=20250415T072206, end_date=20250415T092445
[2025-04-15T09:24:45.770+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-15T09:24:45.788+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-15T09:24:45.799+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
